style.css

```diff
@@ -857,6 +857,29 @@ html[lang="th"] textarea {
 	line-height: 1.8;
 }
 
+/* Remove letter-spacing for all non-latin alphabets */
+
+html[lang="ar"] *,
+html[lang="ary"] *,
+html[lang="azb"] *,
+html[lang="haz"] *,
+html[lang="ps"] *,
+html[lang^="zh-"] *,
+html[lang="bg-BG"] *,
+html[lang="ru-RU"] *,
+html[lang="uk"] *,
+html[lang="bn-BD"] *,
+html[lang="hi-IN"] *,
+html[lang="mr-IN"] *,
+html[lang="el"] *,
+html[lang="gu-IN"] *,
+html[lang="he-IL"] *,
+html[lang="ja"] *,
+html[lang="ko-KR"] *,
+html[lang="th"] * {
+	letter-spacing: normal !important;
```

Comment:
In latin alphabets, the letter spacing creates more breathability in the typeface. In non-latin alphabets, the spacing doesn't work as well, especially alphabets like Devanagari or Arabic that blend letters together like ligatures.

When I first looked at this issue, I tried your approach of finding all of the places where letter-spacing occurs and got more than halfway through before realizing just how ridiculous the amount of CSS was. Using `*` seems much more practical. 

If there's a way I can make the list more specific without using ~!important`, that would be great. If there isn't, I think we should go with it.